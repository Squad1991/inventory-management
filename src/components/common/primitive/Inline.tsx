import { styled } from '@compiled/react';

const Inline = styled.div({
  display: 'flex',
  flexDirection: 'row',
});

export default React.memo(Inline);
