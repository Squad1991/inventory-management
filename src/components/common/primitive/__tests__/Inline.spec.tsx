import { render } from '@testing-library/react';
import React from 'react';
import Inline from '~src/components/common/primitive/Inline';

describe('Inline component', () => {
  it('should match the snapshot', () => {
    const { container } = render(
      <Inline>
        <div>1</div>
        <div>2</div>
        <div>3</div>
      </Inline>,
    );
    expect(container).toMatchSnapshot();
  });
});
