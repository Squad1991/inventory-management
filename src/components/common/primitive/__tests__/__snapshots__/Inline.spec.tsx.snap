// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Inline component > should match the snapshot 1`] = `
<div>
  <style
    data-cmpld="true"
  >
    ._1e0c1txw{display:flex}._2lx2vrvc{flex-direction:row}
  </style>
  <div
    class="_1e0c1txw _2lx2vrvc"
  >
    <div>
      1
    </div>
    <div>
      2
    </div>
    <div>
      3
    </div>
  </div>
</div>
`;
